src/sentry/workflow_engine/migration_helpers/issue_alert_conditions.py

```diff
@@ -153,3 +154,22 @@ def create_tagged_event_data_condition(
         condition_result=True,
         condition_group=dcg,
     )
+
+
+@data_condition_translator_registry.register(AgeComparisonFilter.id)
+def create_age_comparison_data_condition(
+    data: dict[str, Any], dcg: DataConditionGroup
+) -> DataCondition:
+    # TODO: Add comparison validation (error if not enough information)
+    comparison = {
+        "comparison_type": data["comparison_type"],
+        "value": data["value"],
+        "time": data["time"],
```

Comment:
we probably should think about how to validate that this is 1 of the possible choices too for workflow engine APIs. just a thought